icon-exclaim/icon-dependencies-mpich/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# syntax=docker/dockerfile:1.2
+FROM nvcr.io/nvidia/nvhpc:21.3-devel-cuda_multi-ubuntu20.04
+ENV SPACK_PREFIX=/root/c2sm-spack
+ENV SPACK_SYSTEM_CONFIG_PATH=/root/c2sm-spack/sysconfigs/unknown
+ENV SPACK_ROOT=/root/c2sm-spack/spack
+
+RUN apt-get -yqq update \
+ && apt-get -yqq install --no-install-recommends \
+        build-essential \
+        ca-certificates \
+        curl \
+        gnupg2 \
+        iproute2 \
+        locales \
+        python3 \
+        python3-pip \
+        python3-setuptools \
+        tcl \
+        unzip \
+ && locale-gen en_US.UTF-8 \
+ && pip3 install boto3 \
+ && rm -rf /var/lib/apt/lists/*
+
+# setup spack-c2sm instance
+RUN git clone --depth 1 --recurse-submodules --shallow-submodules -b dev_v0.18.1 https://github.com/C2SM/spack-c2sm.git ${SPACK_PREFIX}
+
+# add additional packages from Sergey
+COPY packages/icon ${SPACK_PREFIX}/packages/icon
+COPY packages/rttov ${SPACK_PREFIX}/packages/rttov
+COPY packages/libcdi-pio ${SPACK_PREFIX}/packages/libcdi-pio
+COPY packages/yaxt ${SPACK_PREFIX}/packages/yaxt
+COPY packages/yac ${SPACK_PREFIX}/packages/yac
+COPY packages/scales-ppm ${SPACK_PREFIX}/packages/scales-ppm
+
+# What we want to install and how we want to install it
+# is specified in a manifest file (spack.yaml)
+RUN mkdir /opt/spack-environment \
+&&  (echo "spack:" \
+&&   echo "  specs:" \
+&&   echo "  - eccodes@2.25.0%nvhpc +aec~fortran~ipo~memfs~netcdf~openmp~png~pthreads~python+shared~tools build_type=RelWithDebInfo definitions=auto jp2k=openjpeg samples=auto" \
```

Comment:
This is probably the wrong eccodes version.

It needs to be eccodes@2.19.0 otherwise some of the grib-testcases fail.